chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SchedulerController.java

```diff
@@ -40,7 +40,8 @@
      * @throws SchedulerException if exception is thrown
      * @throws ValidationException if job values aren't correct
      */
-    @ApiOperation(value = "Create a new trigger and return it")
+    @ApiOperation(value = "Create a new trigger and return it",
+            notes = "Security Restrictions: ROLE_ADMIN or ROLE_ACB and have administrative authority on the specified ACB.")
```

Comment:
This one should have ROLE_ONC too, right? Also think there's something odd about the ACB, because not all triggers specify an ACB. Similar comments to the other endpoints related to triggers & ACB

Reply:
Addressed